etl/extractor/mainnet_test.go

```diff
@@ -3,6 +3,8 @@
 // This material is licensed under the Insolar License version 1.0,
 // available at https://github.com/insolar/block-explorer/blob/master/LICENSE.md.
 
+// +build integration
```

Comment:
it's a unit test

Reply:
if I declare `mainnet_test.go` as `unit` it won't build. 
`mainnet_integration_test.go` is integration, so it's better to set both of them to `integration`

Reply:
`recordStream` is used in both of the test files

Reply:
then we need to add a new test file "helper_test.go" without build tags, transfer common methods there. now both methods able to use some methods